enhancements/cluster-logging/multi-cluster-log-forwarder.md

```diff
@@ -0,0 +1,246 @@
+
+```
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: collect-audit-logs
+    rules:
+    - apiGroups:
+      - "logging.openshift.io"
+      resources:
+      - audit-logs
+      verbs:
+      - collect
+
+```
+This role allows collection of application logs and requires the namespace administor to bind the service account to the role like:
+
+```
+    oc create clusterrolebinding kube-audit-log-collection --clusterrole=collect-audit-logs --serviceaccount=openshift-kube-apiserver:audit-collector
+```
+
+* The namespace administrator creates a **ClusterLogForwarder** CR with a name that matches the service account and the inputs for which they are allowed to collect
+
+```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: ClusterLogForwarder
+    metadata:
+      name: audit-collector
+      namespace: openshift-kube-apiserver
+    spec:
+      pipelines:
+       - inputRefs:
+         - audit
+         outputRefs:
+         - loki
+      outputs:
+      - name: loki
+        type: loki
+        url: https://mycenteralizedserver.some.place
+```
+
+* The namespace administrator creates a **ClusterLogging** CR with a name that matches the service account that only includes a spec for collection
+
+```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: "ClusterLogging"
+    metadata:
+      name: audit-collector
+      namespace: openshift-kube-apiserver
+    spec:
+      collection:
+        type: "vector"
+```
+
+##### Verification and Validations 
+The operator will validate resources upon reconciliation of a **ClusterLogForwarder** and **ClusterLogging** CR.  Failure to meet any of the following conditions will stop the operator from deploying a collector and it will add error status to the resource.
+
+* The **ClusterLogForwarder** CR defines a valid spec
+* There exists a serviceaccount that is named the same as the **ClusterLogForwarder** CR
+* The serviceaccount is bound to clusterroles that allow the input spec of the **ClusterLogForwarder** CR
+* There exists a **ClusterLogging*** CR that is named the same as the **ClusterLogForwarder** CR
+* The **ClusterLogging** CR only defines a valid collection spec.
+
+The previous example identifies a valid **ClusterLogForwarder** CR that specs audit logs that are forwarded to a loki stack.  The following example would be rejected by the operator because it specs collection of application logs but does not have the required role binding:
+
+```yaml
+    apiVersion: "logging.openshift.io/v1"
+    kind: ClusterLogForwarder
+    metadata:
+      name: audit-collector
+      namespace: openshift-kube-apiserver
+    spec:
+      pipelines:
+       - inputRefs:
+         - audit
+         - application
+         outputRefs:
+         - loki
+      outputs:
+      - name: loki
+        type: loki
+        url: https://mycenteralizedserver.some.place
+```
+
+
```

Comment:
Suggest an additional section

#### Use of ClusterLogging

In multiple instance mode the ClusterLogging resource is _optional_. This is the reverse of legacy mode where the CL is required and the CLF is optional. In multiple-instance mode, if a CL exists with the same namespace and name  as the CLF, any relevant entries from the CL will be applied to the behavior of the CLF. Irrelevant entries are ignored.
TODO: list exactly what's relevant/irrelevant, something like:
Relevant:
- ManagementState
- LogStore (defines "default" output for this CLF instance)
- Collection (defines collector type etc. for this CLF instance)
Ignored: Visualization, Curation, Forwarder

The goal here is to allow a smooth transition when we drop the CL in v2, but avoid inventing temporary APIs or alternate ways of configuring the things that are currently configured on the CL which affect the collector.